### DISCUSSION TITLE
Mobile data issues

### DISCUSSION THREAD

**Discussion by HassanDalwi** [ReVanced/revanced-discussions#902] [Question] [answered]
Why my app doesn't work on mobile data. it only work with WiFi. 

I checked phone sittings I wasn't disable mobile data. Maybe there's problem inside the application!!!

**Comment by oSumAtrIX** [comment, marked as answer]
Issue originates from your phone certainly

**Reply by HassanDalwi** [reply]
![Screenshot_20230215-191529.jpg](https://user-images.githubusercontent.com/88087719/219102502-0efca544-edb4-427b-b406-af0eb9eeeabc.jpg)



See like this. However othe YouTube application wirks fine. I don't know the reason

**Reply by oSumAtrIX** [reply]
That's an issue with your phone blocking internet access to the app

**Reply by HassanDalwi** [reply]
> That's an issue with your phone blocking internet access to the app

How to solve the problem

**Reply by oSumAtrIX** [reply]
You gotta ask your provider, or whoever is in charge of the phone and it's configuration

**Reply by reisxd** [reply]
> > That's an issue with your phone blocking internet access to the app
> 
> How to solve the problem 

Don't select "Spoof Wifi Connection" patch.

**Reply by HassanDalwi** [reply]
> > > That's an issue with your phone blocking internet access to the app
> > 
> > How to solve the problem 
> 
> Don't select "Spoof Wifi Connection" patch.

What is this. 
What is patch !! 
Can give pics
